src/Controller/Action/RemoveSelectedProductsFromWishlistAction.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Controller\Action;
+
+use BitBag\SyliusWishlistPlugin\Command\Wishlist\RemoveSelectedProductsFromWishlist;
+use BitBag\SyliusWishlistPlugin\Context\WishlistContextInterface;
+use BitBag\SyliusWishlistPlugin\Form\Type\WishlistCollectionType;
+use BitBag\SyliusWishlistPlugin\Processor\WishlistCommandProcessorInterface;
+use Sylius\Component\Order\Context\CartContextInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Twig\Environment;
+
+final class RemoveSelectedProductsFromWishlistAction
+{
+    private WishlistContextInterface $wishlistContext;
+
+    private CartContextInterface $cartContext;
+
+    private FormFactoryInterface $formFactory;
+
+    private FlashBagInterface $flashBag;
+
+    private UrlGeneratorInterface $urlGenerator;
+
+    private Environment $twigEnvironment;
+
+    private MessageBusInterface $commandBus;
+
+    private WishlistCommandProcessorInterface $wishlistCommandProcessor;
+
+    public function __construct(
+        WishlistContextInterface $wishlistContext,
+        CartContextInterface $cartContext,
+        FormFactoryInterface $formFactory,
+        FlashBagInterface $flashBag,
+        UrlGeneratorInterface $urlGenerator,
+        Environment $twigEnvironment,
+        MessageBusInterface $commandBus,
+        WishlistCommandProcessorInterface $wishlistCommandProcessor
+    ) {
+        $this->wishlistContext = $wishlistContext;
+        $this->cartContext = $cartContext;
+        $this->formFactory = $formFactory;
+        $this->flashBag = $flashBag;
+        $this->urlGenerator = $urlGenerator;
+        $this->twigEnvironment = $twigEnvironment;
+        $this->commandBus = $commandBus;
+        $this->wishlistCommandProcessor = $wishlistCommandProcessor;
+    }
+
+    public function __invoke(Request $request): Response
```

Comment:
Too many responsibilities for a single method. Please, look at the SOLID materials and use this knowledge to simplify class.

Reply:
`AddProductsToWishlistAction` contains pretty similar logic, unify please.